app/views/core/SubscribeModal.coffee

```diff
@@ -92,6 +93,14 @@ module.exports = class SubscribeModal extends ModalView
     super()
     # TODO: does this work?
     @playSound 'game-menu-open'
+    if @basicProduct and @subModalContinue
+      if @subModalContinue is 'monthly'
+        @subModalContinue = null
+        @onClickPurchaseButton()
+      else if @subModalContinue is 'lifetime'
+        @subModalContinue = null
+        unless @basicProduct.isRegionalSubscription()
```

Comment:
Could you add a comment as to why this line is here? I don't know off the top of my head.